dev_envs/mac-env-setup.md

```diff
@@ -217,7 +220,8 @@ git config --global commit.gpgsign true
 
 Reference: [GitHub: Generating a new SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent)
 
-Note: use `ed25519` instead of `rsa`
+> **Note**
```

Comment:
Thoughts on using an alternate display for these since it's more of a mandate than a "just for your informatino" item?

```suggestion
> **Warning**
```